apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/main/kotlin/ch/tutteli/atrium/api/fluent/en_GB/anyAssertions.kt

```diff
@@ -30,22 +30,56 @@ fun <T> Expect<T>.notToBe(expected: T): Expect<T> = _logicAppend { notToBe(expec
 /**
  * Expects that the subject of the assertion is the same instance as [expected].
  *
+ * Deprecated as atrium moves to a consistent ‘to + infinitive’ naming convention. Use [toBeTheSameAs] instead.
+ * This function will be removed in version 1.0.0. See
+ * [atrium-roadmap#93](https://github.com/robstoll/atrium-roadmap/issues/93) for details and to give feedback.
+ *
  * @return An [Expect] for the current subject of the assertion.
  * @throws AssertionError Might throw an [AssertionError] if the assertion made is not correct.
+ */
+@Deprecated(
+    "Replaced by toBeTheSameAs. Will be removed with version 1.0.0",
+    ReplaceWith("this.toBeTheSameAs<T>(expected)")
+) // TODO remove with 1.0.0
+fun <T> Expect<T>.isSameAs(expected: T): Expect<T> = toBeTheSameAs(expected)
+
+/**
+ * Expects that the subject of `this` assertion is the same instance as [expected].
  *
- * @sample ch.tutteli.atrium.api.fluent.en_GB.samples.AnyAssertionSamples.isSameAs
+ * @return an [Expect] for the current subject of `this` assertion.
```

Comment:
Thanks for all the little improvements :+1: 
Same problematic as above with `this` and maybe it would also make sense to remove `current`? So my suggestion including your improvement:
`@return An [Expect] for the subject of this assertion.`

Reply:
I am very happy to leave out ‘current’.

I would like to keep the lowercase ‘an’ since what follows is a phrase started by the word ‘return’. To capitalize, it should be a proper sentence (see also the [Javadoc convention for @param](https://www.oracle.com/de/technical-resources/articles/java/javadoc-tool.html#@param)

Reply:
So:
```
@return an [Expect] for the subject of `this` expectation.
```

Reply:
going to address this now.